### PR TITLE
Adding an https-url import definition example to import a large impor…

### PR DESCRIPTION
**Pull request by DanielePalaia**
…t definition at boot time from the web

This closes #

**Note to reviewers:** remember to look at the commits in this PR and consider if they can be squashed

## Local Testing

Please ensure you run the unit, integration and system tests before approving the PR.

To run the unit and integration tests:

```
$ make unit-tests integration-tests
```

You will need to target a k8s cluster and have the operator deployed for running the system tests.

For example, for a Kubernetes context named `dev-bunny`:
```
$ kubectx dev-bunny
$ make destroy deploy-dev
# wait for operator to be deployed
$ make system-tests
```

### CODE REVIEW

**Review by mkuratczyk — APPROVED** [review]
Please remove the `NodePort` section. Otherwise, it's great.

File: docs/examples/import-https-url-definition/rabbitmq.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  name: import-https-url-definitions 
+spec:
+  replicas: 1
+  resources:
+    requests:
+      cpu: 2
+      memory: 8Gi
+    limits:
+      cpu: 2
+      memory: 8Gi
```
Comment (ChunyiLyu) [inline docs/examples/import-https-url-definition/rabbitmq.yaml:13]: Are the resource request and limit required for this example to work? If not, lets remove it to keep the example simple and just about the importing definition itself.

File: docs/examples/import-https-url-definition/rabbitmq.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  rabbitmq:
+    additionalConfig: |
+      definitions.import_backend = https
+      definitions.https.url = https://raw.githubusercontent.com/rabbitmq/sample-configs/main/lot-of-queues/5k-queues.json
+      definitions.tls.versions.1 = tlsv1.2
+      vm_memory_high_watermark_paging_ratio = 0.85
```
Comment (ChunyiLyu) [inline docs/examples/import-https-url-definition/rabbitmq.yaml:19]: Similar question to above, is configuring memory high water mark necessary here?

**Review by ChunyiLyu — COMMENTED** [review]
Please see comments about resource limit and request and the memory high water mark configurations. If they are not required for the example, let's remove it. If they are indeed required, may I suggest adding comments in the example or the README to explain what they are for?

**Comment by DanielePalaia** [comment]
@ChunyiLyu thanks for your review, indeed I was planning to write the motivation in the README but I then forgot. The example takes a lot of memory to load a very big definition file, so I think better to put the resource requests/limits. I removed the vm_memory_high_watermark_paging_ratio as it wasn't really necessary for the example and I updated the README file.

**Review by ChunyiLyu — APPROVED** [review]

**Pull request merged**